examples/kubernetes/image/docker/run.sh

```diff
@@ -1,52 +0,0 @@
-#!/bin/bash
-
-function setup() {
-  # use hostname command to get our pod's ip until downward api are less racy (sometimes the podIP from downward api is empty)
-  export POD_IP=$(hostname -i)
-}
-
-function checkdata() {
-  if [[ ! -e /stolon-data ]]; then
```

Comment:
This check, warning and data creation were removed but it'll be worth keeping them. The warning is not completely correct since already having a /stolon-data doesn't mean that it'll be persisted (it should be mounted on an external volume and that volume should be a k8s persistent volume) but I'm not sure the check should go so deeper.